lib/widgets-page.php

```diff
@@ -41,7 +41,7 @@ function gutenberg_widgets_init( $hook ) {
 		wp_enqueue_style( 'wp-block-library-theme' );
 		wp_add_inline_style(
 			'wp-block-library-theme',
-			'.block-widget-form .widget-control-save { display: none; }'
+			'.wp-block-textarea { width: 100%; min-height: 5em; margin: 8px 0 16px 0; }'
```

Comment:
The class `wp-block-textarea` is what the CSS class for a Textarea block would be, just as e.g. `wp-block-image` is the CSS class for the Image block. Let's use something like `block-widget-textarea` instead.